### DISCUSSION TITLE
Residual errors with LASSO and ENET

### DISCUSSION THREAD

**Discussion by RicardoCecconello** [ttadano/alamode#200] [Q&A] [answered]
Hello there!

How the residual errors (with LASSO and ENET) are calculated? Is it in the same way as for OLS (https://alamode.readthedocs.io/en/latest/almdir/formalism_alm.html#equation-fitting-error)?

Best regards,
Ricardo

**Comment by ttadano** [comment, marked as answer]
Yes, the same definition is used for LASSO and ENET.
